### ISSUE TITLE
Fix CUE Get started with GitLab CI

### ISSUE THREAD

**Issue by thomasboni** [dagger/dagger#4093]
### What is the issue?

The getting started for Dagger CUE on GitLab CI doesn't work.

In the `before_script` the dagger CLI is installed instead of dagger-cue CLI:

```yaml
  before_script:
    - |
      # install dagger
      cd /usr/local
      wget -O - https://dl.dagger.io/dagger/install.sh | sh
      cd -

      dagger-cue version
```

To fix it, I'm going to open a PR where :
- I replaced `https://dl.dagger.io/dagger/install.sh` by `https://dl.dagger.io/dagger-cue/install.sh` 
- I specify a variable `VERSION` to dagger-cue installation script because it doesn't support `DAGGER_VERSION` variable

But, I think it is better to update the dagger-cue install script to use variable `DAGGER_VERSION` in order to have consistent variables naming in installation scripts. Additionally, the script should fetch the last version if I don't specify `VERSION` variable but it doesn't work. I'm going to open another PR dedicated to this update.

**Comment by thomasboni** [comment]
I didn't find where bash installation script of `dagger-cue` is located in the repo. Is it versioned here ?

**Comment by thomasboni** [comment]
Resolved in #4094
